eclipsePlugin/build.gradle

```diff
@@ -19,6 +19,11 @@ sourceSets {
       include '**/*.xml'
     }
   }
+  test {
```

Comment:
please undo this change and enable tests plugin code which is in a dedicated tests plugin.